### PR TITLE
DBZ-3967 Fix issue with removal of default properties

### PR DESCRIPTION
**Pull request by mdrillin**
Fixes issue with removal properties that have defaults.  The check in Utils.minimizePropertyValues was incorrect.

### CODE REVIEW

File: ui/packages/ui/src/app/shared/Utils.ts [hunk 1]
```diff
@@ -349,7 +349,7 @@ export function minimizePropertyValues (propertyValues: Map<string, string>, pro
       if (propDefn.isMandatory) {
         minimizedValues.set(key, value);
       // Include non-mandatory if has default and value is different
-      }else if (propDefn.defaultValue) {
+      }else if (propDefn.defaultValue != null) {
```
Comment (rk3rn3r) [inline ui/packages/ui/src/app/shared/Utils.ts:352]: Hey @mdrillin can you maybe have a look into the backend code and fix sending any null to the UI too with that PR?

Comment (mdrillin) [reply ui/packages/ui/src/app/shared/Utils.ts:352]: Sure, but I'd suggest working that as a separate PR.  Is there a JIRA for it?

Comment (rk3rn3r) [reply ui/packages/ui/src/app/shared/Utils.ts:352]: You can just do it here in this PR. The purpose is the same.

Comment (mdrillin) [reply ui/packages/ui/src/app/shared/Utils.ts:352]: The purpose is not the same.  This PR eliminates sending of properties with default values from the UI to the backend.

Comment (rk3rn3r) [reply ui/packages/ui/src/app/shared/Utils.ts:352]: That's a question of the pov. "Fix issue with removal of default properties" can also mean: one side is fixing issues with such data in the frontend, but can also cover eliminating sending the wrong data in the first place from the backend.

**Comment by mdrillin** [comment]
@indraraj @uidoyen - this can be reviewed and merged.  The other issue in question was resolved separately.  DBZ-3967 gives an example of the properties affected by this PR.

**Review by indraraj — APPROVED** [review]
Tested locally, looks good.

**Pull request merged**